education/demo/student_demo.xml

```diff
@@ -348,7 +331,6 @@
             <field name="last_name">Bennett</field>
             <field name="gender">f</field>
```

Comment:
We have a `gender` in [`partner_contact_gender`](https://github.com/OCA/partner-contact/tree/10.0/partner_contact_gender), although we'd need to ask for a license change to include here.